app/views/shared/_flash_messages.html.erb

```diff
@@ -1,2 +1,6 @@
-<p class="notice"><%= notice %></p>
-<p class="alert"><%= alert %></p>
+<% if notice %>
```

Comment:
Indentation
```erb
<% if notice %>
  <p class="alert alert-success mt-3"><%= notice %></p>
<% end %>
<% if alert %>
  <p class="alert alert-danger mt-3"><%= alert %></p>
<% end %>

```

Reply:
@junan Nice catch! What IDE are you using? I'm using VSCode + Beautify to format, seemed like the Beautify tool format `.erb` with the wrong indentation?

Reply:
@suho For Ruby project, I use [Rubymine](https://www.jetbrains.com/ruby/)

Reply:
Fixed in [`5516c98` (#31)](https://github.com/suho/google-search-ruby/pull/31/commits/5516c985d08f72d9f9e17220cbc82f88ee49cb30)